src/applications/mhv-secure-messaging/tests/e2e/fixtures/pilot-responses/inbox-threads-OH-response.json

```diff
@@ -0,0 +1,85 @@
+{
+  "data": [
+    {
+      "id": "3763993",
+      "type": "message_threads",
+      "attributes": {
+        "threadId": 3763993,
+        "folderId": 0,
+        "messageId": 3764003,
+        "threadPageSize": 3,
+        "messageCount": 5,
+        "category": "OTHER",
+        "subject": "OH THREAD 002 ATTACHMENT",
+        "triageGroupName": "OH TG GROUP 002 ",
+        "sentDate": "2024-07-10T18:53:28.000Z",
+        "draftDate": null,
+        "senderId": 2444644,
+        "senderName": "HAMLAT, ABDELHALIM",
```

Comment:
May need to change this PII with fake data. there is more below

Reply:
done